osu.Game.Rulesets.Catch.Tests/TestSceneCatcherArea.cs

```diff
@@ -47,7 +85,8 @@ private void createCatcher(float size)
                 Child = new TestCatcherArea(new BeatmapDifficulty { CircleSize = size })
                 {
                     Anchor = Anchor.CentreLeft,
-                    Origin = Anchor.TopLeft
+                    Origin = Anchor.TopLeft,
+                    CreateDrawableRepresentation = ((DrawableRuleset<CatchHitObject>)catchRuleset.CreateInstance().CreateDrawableRulesetWith(new CatchBeatmap())).CreateDrawableRepresentation
```

Comment:
That's a *bit* of a line, but it's test code so doesn't matter too much, I suppose.